app/AppKernel.php

```diff
@@ -35,11 +35,19 @@ public function getRootDir()
 
     public function getCacheDir()
     {
+        if (in_array($this->environment, ['dev', 'test'])) {
+            return '/dev/shm/symfony_docker_test/cache/'.$this->environment;
+        }
+
         return dirname(__DIR__).'/var/cache/'.$this->getEnvironment();
     }
 
     public function getLogDir()
     {
+        if (in_array($this->environment, ['dev', 'test'])) {
```

Comment:
@michaelperrin should not be environment dependent in fact... running a symfony app in docker, you should NEVER write logs in local files in fact, you should log everything on stdout & use something like logspout or docker log drivers to aggregate logs where you want ;)